src/js/Nittro/Page/Service.js

```diff
@@ -259,7 +259,8 @@ _context.invoke('Nittro.Page', function (DOM, Arrays, Url, SnippetHelpers, Snipp
             var request = this._.ajax.createRequest(url, method, data);
 
             try {
-                var p = this._dispatchRequest(request, context, true);
+                var pushState = DOM.getData(context, 'push-state');
```

Comment:
`context` can be empty here, e.g. if the request was triggered from a custom script